metricbeat/mb/registry.go

```diff
@@ -222,16 +236,22 @@ func (r *Register) metricSetRegistration(module, name string) (MetricSetRegistra
 	name = strings.ToLower(name)
 
 	metricSets, exists := r.metricSets[module]
-	if !exists {
-		return MetricSetRegistration{}, fmt.Errorf("metricset '%s/%s' is not registered, module not found", module, name)
+	if exists {
+		registration, exists := metricSets[name]
+		if exists {
+			return registration, nil
+		}
 	}
 
-	registration, exists := metricSets[name]
-	if !exists {
-		return MetricSetRegistration{}, fmt.Errorf("metricset '%s/%s' is not registered, metricset not found", module, name)
+	if source := r.secondarySource; source != nil && source.HasMetricSet(module, name) {
```

Comment:
a comment here would help to understand this is a fallback

Reply:
Ok, I will add something to these blocks.